app/views/groups/show.html.erb

```diff
@@ -52,19 +58,28 @@
       <%= link_to t("groups.show.assignment.add_assignment_btn"), new_group_assignment_path(@group), class: "btn primary-button groups-primary-button" %>
     <% end %>
   </div>
-  <hr>
+    <% if @group.assignments.empty? %>
+      <hr>
+      <% if @group.assignments.empty? %>
+        <span class="null-text"><%= t("groups.show.no_assignments_html") %></span>
+      <% end %>
+    <% end %>
```

Comment:
What is the meaning of adding this again?
This renders the warning twice!

Reply:
😅 forgot to test ... blindly committed code review suggestions. Sorry for inconvenience.